src/FAudio_platform_win32.c

```diff
@@ -41,6 +41,7 @@
 #include <initguid.h>
 #include <audioclient.h>
 #include <mmdeviceapi.h>
+#include <Functiondiscoverykeys_devpkey.h>
```

Comment:
Could you use `devpkey.h` and then, `DEVPKEY_Device_FriendlyName` instead? Fwiw, `Functiondiscoverykeys_devpkey.h` doesn't exist in Wine.

Reply:
Done.